build-gluster-org/jobs/centos7-regression.yml

```diff
@@ -1,40 +1,73 @@
 - job:
-    name: centos7-regression
+    name: gh_centos7-regression
     node: centos7
     description: Our production regression test.  Triggerered for each Gerrit CR revision.
     project-type: freestyle
     concurrent: true
 
     scm:
-    - git:
-        branches:
-        - $GERRIT_BRANCH
-        refspec: $GERRIT_REFSPEC
-        choosing-strategy: gerrit
-        url: git://review.gluster.org/glusterfs.git
-        wipe-workspace: false
+    - glusterfs
 
     properties:
+    - github:
+        url: https://github.com/gluster/glusterfs
     - discard-after-x:
         x: 90
     - one-build-per-node
 
     triggers:
-    - gerrit:
-        server-name: review.gluster.org
-        trigger-on:
-          - comment-added-contains-event:
-              comment-contains-value: "recheck centos"
-          - comment-added-event:
-              approval-category: Verified
-              approval-value: 1
+    - github-pull-request:
+        cancel-builds-on-update: true
+        allow-whitelist-orgs-as-admins: true
+        admin-list:
+        # This list is extracted from the Github's gluster.org team "gluster-maintainers".
+        # There's no automation of syncing the list with github's team. We need to manually add the person.
+          - deepshikhaaa
+          - mscherer
+          - csabahenk
+          - jdarcy
+          - itisravi
+          - aravindavk
+          - kalebskeithley
+          - nigelbabu
+          - nixpanic
+          - prashanthpai
+          - pranithk
+          - raghavendrabhat
+          - vbellur
+          - amarts
+          - raghavendrahg
+          - raghavendra-talur
+          - KritikaDhananjay
+          - thotz
+          - rafikc30
+          - humblec
+          - poornimag
+          - ShyamsundarR
+          - anoopcs9
+          - soumyakoduri
+          - spalai
+          - atinmu
+          - kotreshhr
+          - mchangir
+          - nbalacha
+          - mohit84
+          - sunnyku
+          - sanjurakonde
+          - xhernandez
+        org-list:
+          - gluster
+        github-hooks: true
+        only-trigger-phrase: true
+        trigger-phrase: '/run regression'
+        skip-build-phrase: 'no tests'
+        permit-all: true
```

Comment:
I think that's not what we want: "permit-all (bool) – build every pull request automatically without asking (default false)"

https://jenkins-job-builder.readthedocs.io/en/latest/triggers.html